forge/src/main/java/org/openjdk/skara/forge/github/GitHubPullRequest.java

```diff
@@ -783,4 +787,15 @@ public Optional<Hash> findIntegratedCommitHash() {
     public Object snapshot() {
         return json;
     }
+
+    public String limitBodySize(String body) {
+        if (body.length() > GITHUB_PR_COMMENT_BODY_MAX_SIZE) {
+            return "⚠️This comment is too long, only the first "
```

Comment:
I think ending with `...` is enough to indicate that the comment has been truncated. This will only apply to comments generated by the bot. If we start such a comment with a warning like this, it will risk confusing users.